multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -91,3 +102,54 @@ storage:
 
             echo "Importing tarball into volume"
             sudo -u builder -H /bin/bash -c "cd /var/home/builder; podman volume import secex-data /var/home/builder/${TARBALL}"
+
+    - path: /usr/local/bin/create-cex-config.sh
+      mode: 0755
+      user:
+        name: root
+      group:
+        name: root
+      contents:
```

Comment:
Let's make this script live outside the Butane config and then use `local` to feed it to Butane (see third example in https://coreos.github.io/butane/examples/#files).

Reply:
Hi @jlebon ,  Sorry i did not understand . Where do we keep the script in local ? if we are executing in a shell with butane command we can use -d to take the relative path of the script. how we can use here?

Reply:
You can keep it in the same `multi-arch-builders/` directory in this repo.